virtcontainers/sandbox.go

```diff
@@ -127,7 +127,10 @@ type SandboxConfig struct {
 
 	// Cgroups specifies specific cgroup settings for the various subsystems that the container is
 	// placed into to limit the resources the container has available
-	Cgroups *configs.Cgroup `json:"cgroups"`
+	Cgroups *configs.Cgroup
+
+	// CRIContainerType specifies whether container type was set explicitly through annotations or not.
+	CRIContainerType bool
```

Comment:
Nit: It might be worth renaming this variable to make it slightly clearer as on first inspection is seems like it's specifying a type for a container (which it isn't ;)